src/main/kotlin/com/faendir/intellij/gradleVersionCatalogs/kotlin/Accessor.kt

```diff
@@ -15,11 +15,18 @@ private const val BUNDLE_SUPPLIER = "org.gradle.api.internal.catalog.ExternalMod
 private const val PLUGIN_SUPPLIER = "org.gradle.api.internal.catalog.ExternalModuleDependencyFactory.PluginNotationSupplier"
 
 
+@Suppress("SpellCheckingInspection")
 data class Accessor(val element: PsiElement, val id: String, val type: VCElementType) {
     companion object {
         fun find(element: PsiElement): Accessor? {
-            val returnType = element.lastChild.references.map { it.resolve() }.firstIsInstanceOrNull<PsiMethod>()?.returnType ?: return null
-            val segments by lazy { element.text.replace(Regex("\\s+"), "").split(".").drop(1) }
+            val references = if (element.lastChild.references.mapNotNull(PsiReference::resolve).isEmpty()) {
+                element.lastChild.firstChild.references
+            } else element.lastChild.references
+            val returnType = references.map { it.resolve() }.firstIsInstanceOrNull<PsiMethod>()?.returnType ?: return null
+            val segments by lazy { element.text.replace(Regex("\\s+"), "").split(".").drop(1)
+                // e.g. libs.map.get3dmap() -> [libs, map, get3dmap()] -> [map, get3dmap()] -> [map, 3dmap]
+                .map { it.removePrefix("get").removeSuffix("()") }
```

Comment:
This should only remove the get prefix if it is actually a function call, otherwise libraries starting with get will break.

Reply:
I have tested various use cases such as `libs.map.get3dmap`, `libs.get3dmap().map`, `libs.get3dmap().get2dmap()`.it works great.At the moment I don't know how to write test cases for Inlay Hint. Maybe later I will actively look for options to write unit test cases.

Reply:
I made some adjustments and it's working properly.
![WX20230725-182857](https://github.com/F43nd1r/gradle-version-catalogs-intellij-plugin/assets/6847158/693f0a4e-dfb4-4a11-84f3-73dc3a767244)
![WX20230725-182357](https://github.com/F43nd1r/gradle-version-catalogs-intellij-plugin/assets/6847158/c916505e-0fc1-40ff-aea5-bbde36a35d70)